### ISSUE TITLE
[Bug]: Description meta is lost when expanding composite tokens

### ISSUE THREAD

**Issue by jorenbroekema** [tokens-studio/sd-transforms#103]
### What happened?

Description property meta is lost when expanding composite tokens, meaning you don't get a comment in the output CSS.

There are probably two ways to go about it:
- Copy over the meta into the separate expanded tokens, so they all share a copy of the same meta. This seems like the better solution for "any" generic metadata
- Put it on the first property only, which might be nicer for something like description, because you wouldn't really wanna see the same comment 4+ times?

### Reproduction

[Configurator Link](https://configurator.tokens.studio/#project=rVVNb9NAEP0rIwspSeXEpUI9JBSBqEqPiCA44Ipu7HG8wd61dtclwfJ/Z9brjziBXiCHZDOfb57fjisvkiLh28VOS+EtvSoUAKGnZakiDL0lfLMGa7q4CC4WRv5AoZvg0LOeB98lFBkziVS5tjlVlxPp4//WYhQT2gZ+ULIsrJNsTdG5NmXMZej5Q3ShMOF7F6XjkWtT8iz+yEzqvM3fgPoFo6iEZ6iPxrCfajjakBi14YIZbkeypb4/McXZhhIXDX5/HG/BM+NCbb8+2hHiPnV3fHCH2u8o2f0bI38ce/cfph6G3j0/8449MR0pXpgA9fWzQ9sf+qo931O45dqgWt/elSKybW0jEhzPC6kMrM0hw1veeJg6QKJkDhNtrfO4N09WoWgTKuhKfu4I1FC3eW9HBAY6nvcsa1sjFEEAI6sPVyKGgimWI9UEMoIsGpwQMQEbBBbHGDeZa0SIZaSXkBpT6GUQbLlJy80iknnwTOtQnGOenkzu02S4L5iISSVgDoXcKlakhyUYVSLU9azDj3uWFxnCE8vI0Xfx4WfKoxR2pTbEkSkVjWBSirDAgGngBOSk6+IM2LQRDPVHpw7q0cpCEEXWlh/eUwuZ9ymtn5QSpaiWMG06zuDmDVTuAvQgT91OMA6tA7oCGvH1fD7kWOgCsXkITlezlgknT0gxK1Bpp4FT6YSCFp22urG34x5ZjMpvMw3GXzr5k4ZuTtW4cGH3rv4p/7QE2jp/p/Wu8TtOHX+Tjj7nmjT8dHCInaS9JtMqPtKGxe53wqxnHXHdaLI0RWk+YYIKRdQO00avRhw/vqgGHqbNuZ7VSyWlsUXJe8bMtLURdhp54sMxsNPOVIwe0OOqf1R0ojXgbsXxa2R45yRSmHnOuBi9QlrljVamjbxjOc8ObiW9I4TZeANSxFfk27RdWteXlyN/xgXNPvivLs/S1/wXOufLV8W+23PDIrdXo93W/SXtizQLtlmS/YJdy5y0ImUGI1e/I+vf)

```css
:root {
  --sdFontMainFontFamily: Arial;
  --sdFontMainFontWeight: 600;
  --sdFontMainLineHeight: 20;
  --sdFontMainFontSize: 14px;
}
```

### Expected output

```css
:root {
   /* Some cool description */
  --sdFontMainFontFamily: Arial;
  --sdFontMainFontWeight: 600;
  --sdFontMainLineHeight: 20;
  --sdFontMainFontSize: 14px;
}
```
Which at the moment cannot really be accomplished without https://github.com/amzn/style-dictionary/pull/953, and you'd have to put a newline in the description somewhere to manipulate it into placing it above the token. So perhaps it would be cool if that was a commentStyle option as well (short-above, long-above, in addition to short/long, where above is just forced for comments with newlines)

For now, 

```css
:root {
  --sdFontMainFontFamily: Arial; /* Some cool description */
  --sdFontMainFontWeight: 600;
  --sdFontMainLineHeight: 20;
  --sdFontMainFontSize: 14px;
}
```
would already be better, so just moving the description of the group to the first property of the expanded parts is a good intermediate solution

Or alternatively, no special case for description, copying the meta to all props:
```css
:root {
  --sdFontMainFontFamily: Arial; /* Some cool description */
  --sdFontMainFontWeight: 600; /* Some cool description */
  --sdFontMainLineHeight: 20; /* Some cool description */
  --sdFontMainFontSize: 14px; /* Some cool description */
}
```

### Version

0.8.1

**Comment by jorenbroekema** [comment]
@six7 would love your feedback on this.

**Comment by six7** [comment]
Let's do version A: Copy over the meta into the separate expanded tokens, so they all share a copy of the same meta. This seems like the better solution for "any" generic metadata

I feel that way you can look at a token, no matter which part of a composite token it is, and understand the added context.

**Comment by thomasmattheussen** [comment]
Just to add to this: 

```
:root {
  --sdFontMainFontFamily: Arial; /* Some cool description */
  --sdFontMainFontWeight: 600; /* Some cool description */
  --sdFontMainLineHeight: 20; /* Some cool description */
  --sdFontMainFontSize: 14px; /* Some cool description */
}
```

this seems (currently) a better solution since style-dictionary output isn't always deterministic in it's output ordering. These tokens may not always be grouped together.

**Comment by jorenbroekema** [comment]
Fixed in v0.14.1
